src/adapters/Logary.Adapters.AspNetCore/Extensions.fs

```diff
@@ -0,0 +1,25 @@
+namespace Logary.Adapters.AspNetCore
+
+open Logary.Configuration
+open Microsoft.Extensions.Logging
+open System.Runtime.CompilerServices
+
+
+[<AutoOpen;Extension>]
+module ILoggingBuilderExtensions =
+
+    [<Extension>]
```

Comment:
Better to use `CompiledName` here, than to use a non-standard method signature for this project; that way, if you use this code from C#, you'll have the NameCasedLikeThis.

Reply:
got you. in pr #384 fixed